### PR TITLE
Parallel_analyses

### PR DESCRIPTION
**Pull request by PerryShafran-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

These changes have been running in parallel in the emc.vpppg group account for awhile and works well.

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

Run the scripts as you normally would. Check the output to compare to your own runs. You may need to link to the /lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0 output if your runs need several days worth of data. You may recommend merging into the auth repo but you do not have the authority to perform the merge.

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

At least 5 business days from receipt of this PR.

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

File: ush/analyses/plot_util.py [hunk 1]
```diff
@@ -1624,7 +1624,7 @@ def get_ci_file(stat, input_filename, fcst_lead, output_base_dir, ci_method):
    return CI_file          
 
 def equalize_samples(logger, df, group_by):
-    with open('/u/perry.shafran/test.pkl','wb') as f:
+    with open('/u/emc.vpppg/test.pkl','wb') as f:
```
Comment (ShelleyMelchior-NOAA) [inline ush/analyses/plot_util.py:1627]: @PerryShafran-NOAA -- one question here. What happens in this script when the job is run by NCO? How can we make this more flexible?

**Review by ShelleyMelchior-NOAA — COMMENTED** [review]
See comment in `ush/analyses/plot_util.py`.

File: ush/analyses/plot_util.py (hunk 1 above)
Comment (PerryShafran-NOAA) [reply ush/analyses/plot_util.py:1627]: I think that Marcel had changed that somewhere.  We'll need to corroborate fixes with him at some point, so this won't be submitted to NCO in this manner.

Comment (ShelleyMelchior-NOAA) [reply ush/analyses/plot_util.py:1627]: @MarcelCaron-NOAA  -- can you suggest the work-around for using the explicit user name in the `plot_util.py` ush script?

**Comment by ShelleyMelchior-NOAA** [comment]
I will proceed w/ approving this PR, understanding that there will need to be future effort to resolve the usage of an explicit username in `ush/analyses/plot_util.py`.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
I will proceed w/ approving this PR, understanding that there will need to be future effort to resolve the usage of an explicit username in `ush/analyses/plot_util.py`.

**Pull request merged**